urdf2webots/parserURDF.py

```diff
@@ -622,7 +675,8 @@ def getVisual(link, node):
             visual.geometry.sphere.radius = float(geometryElement.getElementsByTagName('sphere')[0].getAttribute('radius'))
             link.visual.append(visual)
         elif hasElement(geometryElement, 'mesh'):
-            meshfile = geometryElement.getElementsByTagName('mesh')[0].getAttribute('filename')
+            meshfile = os.path.normpath(os.path.join(path,
+                                                     geometryElement.getElementsByTagName('mesh')[0].getAttribute('filename')))
```

Comment:
Are you sure that this will work if `filename` is an absolute path?

Reply:
I assumed it we never get an absolute path here. Am I wrong?

Reply:
Anyhow I added a test in case we get an absolute path there, but I doubt this will happen.

Reply:
From the documentation nothing prevents to use an absolute path (whereas `obj` file are not supposed to be supported):
> A trimesh element specified by a filename, and an optional scale that scales the mesh's axis-aligned-bounding-box. The recommended format for best texture and color support is Collada .dae files, though .stl files are also supported. The mesh file is not transferred between machines referencing the same model. It must be a local file. 

Furthermore, we might actually get absolute path due to:
https://github.com/cyberbotics/urdf2webots/blob/9b0e41069a8e73e9b518eb97c5f2d2d8deb65963/urdf2webots/importer.py#L73

Using `os.path.relpath` instead looks safer.